pkg/rkenodeconfigserver/nodeserver.go

```diff
@@ -291,3 +294,12 @@ func enhanceWindowsProcesses(processes map[string]v3.Process) map[string]v3.Proc
 
 	return newProcesses
 }
+
+func appendTaintsToKubeletArgs(processes map[string]v3.Process, initialTaints []v3.RKETaint) map[string]v3.Process {
+	if kubelet, ok := processes["kubelet"]; ok && len(initialTaints) != 0 {
+		taintArgs := fmt.Sprintf("--register-with-taints=%s", strings.Join(taints.GetStringsFromRKETaint(initialTaints), ","))
```

Comment:
@superseb can you think of any implications of using this flag on a kubelet start

Reply:
@orangedeng would - and should - user be able to override this flag via passing custom kubelet arguments in cluster.yml config

Reply:
We use the same on controlplane nodes, as long as it can take updates from the Rancher side (override), it should be fine

Reply:
As kubelet can only accept only one `--register-with-taints` parameter, I think we should merge the taints from user custom taints and our taints.